components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3356,6 +3383,7 @@ where
                         state.pending_msgs = drainer.collect();
                         break;
                     }
+                    self.delegate.priority = Priority::Normal;
```

Comment:
This operation will change it to `Normal` priority even if it just processed a high latency command?

Reply:
Yes, restore priority to normal so that other commands can be processed by normal thread.